entrypoint.sh

```diff
@@ -15,4 +15,4 @@ else
 fi
 
 echo "BRANCH_TAG=$BRANCH_TAG" >> $GITHUB_ENV
-echo "::set-output name=branch_tag::$BRANCH_TAG"
+echo "name=branch_tag::$BRANCH_TAG" >> $GITHUB_OUTPUT
```

Comment:
Following the blog documentation, I think it should be the same as declaring ENV:
```suggestion
echo "branch_tag=$BRANCH_TAG" >> $GITHUB_OUTPUT
```

Reply:
You are right. Fixed in [a86dd9c](https://github.com/nimblehq/branch-tag-action/pull/6/commits/a86dd9c4a2a4cf19b770dd001fc2fa7c66014dcf)